scipy/stats/_stats_py.py

```diff
@@ -10115,7 +10111,8 @@ def _sum_of_squares(a, axis=0):
 
     """
     a, axis = _chk_asarray(a, axis)
-    return np.sum(a*a, axis)
+    # return np.linalg.norm(a, axis=axis)**2  # faster, but loses precision
```

Comment:
```suggestion
```

Note that `_chk_asarray` ravels input with `axis=None`, and `axis` tuples aren't supported, so data-apis/array-api#910 doesn't bite us here.